README.md

```diff
@@ -11,9 +11,123 @@ and install the configured UI extension. All configuration is provided
+ extension.version: 'v0.3.1'
+ # optional fields
+ extension.name: 'example'
+ extension.enabled: 'true'
+ extension.checksum_url: 'http://example.com/extension_checksums.txt'
+ extension.max_download_sec: '30'
+ extension.js_vars : |
+     {
+       "key1": "value1",
+       "key2": "value2"
+     }
+
+```
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: argocd-server
+spec:
+  template:
+    spec:
+      initContainers:
+        - name: extension
+          # The image digest must be appended after the tag.
+          # New tag digest can be obtained in quay by clicking in the
+          # "fetch tag" icon and select "Docker Pull (by digest)":
+          # https://quay.io/repository/argoprojlabs/argocd-extension-installer?tab=tags
+          image: docker.intuit.com/quay-rmt/argoprojlabs/argocd-extension-installer:v0.0.1@sha256:f50fa11a4592f3fcdd5a137dab8ed32067bb779a77a393f179e8a5d96abe1a80
+          env:
+          - name: EXTENSION_NAME
+            valueFrom:
+              configMapKeyRef:
+                key: extension.name
+                name: extension-cm
+          - name: EXTENSION_URL
+            valueFrom:
+             configMapKeyRef:
+              key: extension.url
+              name: extension-cm
+          - name: EXTENSION_VERSION
+            valueFrom:
+             configMapKeyRef:
+              key: extension.version
+              name: extension-cm
+          - name: EXTENSION_CHECKSUM_URL
+            valueFrom:
+              configMapKeyRef:
+                key: extension.checksum_url
+                name: extension-cm
+            ## Optional fields
+          - name: $EXTENSION_JS_VARS
+            valueFrom:
+             configMapKeyRef:
+              key: extension.js_vars
+              name: extension-cm
+          volumeMounts:
+            - name: extensions
+              mountPath: /tmp/extensions/
+          securityContext:
+            runAsUser: 1000
+            allowPrivilegeEscalation: false
+      containers:
+        - name: argocd-server
+          volumeMounts:
+            - name: extensions
+              mountPath: /tmp/extensions/
+```
+
+### Exporting env as js extensions
+
+Add the below config in the `extension-cm`:
+```yaml
+#name should match with the extension name e.g 'Metrics', 'Rollout', 'EphemeralAccess'
+extension.name: 'example'
+extension.js_vars : |
+     {
+       "key1": "value1",
+       "key2": "value2"
+     }
+```
+Provide the config in argocd-server deployment as below:
+```yaml
+ ## Optional fields
+  - name: $EXTENSION_JS_VARS
+    valueFrom:
+     configMapKeyRef:
+      key: extension.js_vars
+      name: extension-cm
+```
+output:
+```js
+((window) => {
+    const vars = {
+        "key1": "value1",  "key2": "value2"
+    };
+    window.Extension_VARS = vars;
+})(window);
+```
+Exported envirnoment variables available in the widget:
+![imge](./image/exported_envirnoment_variables.png)
```

Comment:
The variables present in the image don't correspond with the example above which can confuse ppl.
Please update the image with the vars from the example above.

Reply:
update the image